docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3"
+services:
+  redis-server:
+    image: redis
+    hostname: gafm-redis
+    command: ["redis-server", "--save", "60", "1", "--loglevel", "warning"]
+    volumes:
+      - redis-data:/data
+    networks:
+      net:
+        ipv4_address: 172.16.255.10
+    container_name: gafm-redis
+  gafm-service:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    hostname: gafm-srv
+    networks:
+      net:
+        ipv4_address: 172.16.255.11
+    ports:
+      - "443:443" 
+      - "80:443"
+      - "8443:443"
```

Comment:
I don't think this will properly redirect HTTP to HTTPS. You need an NGINX reverse proxy in there to handle the redirection.

Reply:
correct. This was just showing multi port to the same https server